CMake/External_OpenCV.cmake

```diff
@@ -200,6 +236,11 @@ if (CMAKE_COMPILER_IS_GNUCC AND NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6)
   list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_PRECOMPILED_HEADERS:BOOL=OFF)
 endif()
 
+# OpenCV 3.3.0 has an option to enable C++ 11
+if (fletch_BUILD_CXX11)
+  list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DENABLE_CXX11:BOOL=ON)
```

Comment:
Great catch and bad news for us ...  I added that variable as the overall control for CXX_11 in fletch. That really leaves us 2 choices, 1) increase our minimum_cmake_version to 3.3.0 or 2) replace the current CXX_11 control with something less elegant.

I rather prefer the first since we have already discussed raising the CMake version and this gives us a stake in the ground about why we selected the version we did. That said, it is a larger issue which needs more buyin. Leave it for now and I will add an 'issue' to remember the current problem.